### PR TITLE
[[ Feature ]] Redesigned the iOS Standalone Builder Settings Pane

### PR DESCRIPTION
**Pull request by abrisan**

### CODE REVIEW

File: ide-support/revsaveasiosstandalone.livecodescript [hunk 1]
```diff
@@ -1507,13 +1527,35 @@ private command revCreateMobilePlist pSettings, pAppBundle, pTarget, pFonts, pPl
...
+   -- the background modes into the plist file
+   
+   local tBackgroundValues
+   put "<string>audio</string>" into tBackgroundValues["${PLAY_AUDIO_WHEN_IN_BACKGROUND}"]
+   put "<string>location</string>" into tBackgroundValues["${BACKGROUND_LOCATION_UPDATE}"]
+   put "<string>voip</string>" into tBackgroundValues["${BACKGROUND_VOIP}"]
+   put "<string>newsstand-content</string>" into tBackgroundValues["${BACKGROUND_NEWSSTAND_DOWNLOADS}"]
+   put "<string>external-accessory</string>" into tBackgroundValues["${EXTERNAL_ACC_COMM}"]
+   put "<string>bluetooth-central</string>" into tBackgroundValues["${USE_BT_LE}"]
+   put "<string>bluetooth-peripherla</string>" into tBackgroundValues["${ACTS_AS_BT_LE}"]
```
Comment (livecodepanos) [inline ide-support/revsaveasiosstandalone.livecodescript]: This should be `"<string>bluetooth-peripheral</string>"` :)

File: ide-support/revsaveasiosstandalone.livecodescript [hunk 2]
```diff
@@ -1507,13 +1527,35 @@ private command revCreateMobilePlist pSettings, pAppBundle, pTarget, pFonts, pPl
...
-      get "<string>audio</string>"
-   else -- false or empty
-      get empty
-   end if
-   replace "${PLAY_AUDIO_WHEN_IN_BACKGROUND}" with it in pPlist
+   -- AB-2017-05-31: Add BackgroundModes to Plist
+   repeat for each key tKey in tBackgroundModes
+      if tBackgroundModes[tKey] is true then
+         replace tKey with tBackgroundValues[tKey] in pPlist
+      end if
```
Comment (livecodepanos) [inline ide-support/revsaveasiosstandalone.livecodescript]: If for a `tKey` the `tBackgroundModes[tKey]` is not true, you have to remove the `tKey` from the plist:

```
if tBackgroundModes[tKey] is true then
      replace tKey with tBackgroundValues[tKey] in pPlist
else
     replace tKey with empty in pPList
end if
```

**Comment by livecodepanos** [comment]
@livecode-vulcan review ok 075e3f1

**Comment by livecode-vulcan** [comment]
💙 review by @livecodepanos ok 075e3f108dbad5fda571aa209051fa1082185ab0

**Comment by livecode-vulcan** [comment]
😎 test success 075e3f108dbad5fda571aa209051fa1082185ab0

* try-community-armv6-android-api8: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-api8/builds/349)
* try-community-armv6-android-api9: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-api9/builds/355)
* try-community-js-emscripten-sdk1.35: [success](https://vulcan.livecode.com/builders/try-community-js-emscripten-sdk1.35/builds/335)
* try-community-universal-ios-iphoneos10.3: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphoneos10.3/builds/278)
* try-community-universal-ios-iphonesimulator10.3: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphonesimulator10.3/builds/278)
* try-community-universal-mac-macosx10.6: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.6/builds/344)
* try-community-universal-mac-macosx10.9: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.9/builds/349)
* try-community-x86-linux-debian7: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian7/builds/347)
* try-community-x86-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian8/builds/353)
* try-community-x86_64-linux-debian7: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian7/builds/347)
* try-community-x86_64-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian8/builds/353)
* try-community-x86-win32: [success](https://vulcan.livecode.com/builders/try-community-x86-win32/builds/372)
* try-community-x86_64-win32: [success](https://vulcan.livecode.com/builders/try-community-x86_64-win32/builds/343)

**Pull request merged**